packages/abap-deploy-config-sub-generator/src/app/questions.ts

```diff
@@ -0,0 +1,135 @@
+import { getPrompts } from '@sap-ux/abap-deploy-config-inquirer';
+import { FileName, readUi5Yaml } from '@sap-ux/project-access';
+import { getHostEnvironment, hostEnvironment } from '@sap-ux/fiori-generator-shared';
+import { isAppStudio } from '@sap-ux/btp-utils';
+import { ABAP_DEPLOY_TASK } from '../utils/constants';
+import type { ILogWrapper } from '@sap-ux/fiori-generator-shared';
+import type { AbapDeployConfigAnswersInternal, AbapDeployConfigQuestion } from '@sap-ux/abap-deploy-config-inquirer';
+import type { AbapDeployConfig, AbapTarget, BspApp, FioriToolsProxyConfigBackend } from '@sap-ux/ui5-config';
+import type { ConnectedSystem } from '@sap-ux/deploy-config-generator-shared';
+import type { Logger } from '@sap-ux/logger';
+import type { Destination } from '@sap-ux/btp-utils';
+import type { BackendSystem } from '@sap-ux/store';
+import { DeployProjectType } from './types';
+
+/**
+ * Get the ABAP target based on the provided parameters.
+ *
+ * @param destination - the destination used during app generation
+ * @param backendSystem - the backend system used during app generation
+ * @param existingAbapDeployTask - possible existing deploy configuration (ui5-deploy.yaml)
+ * @param backendConfig - the existing base configuration (ui5.yaml)
+ * @returns - the ABAP target
+ */
+function getAbapTarget(
+    destination?: Destination,
+    backendSystem?: BackendSystem,
+    existingAbapDeployTask?: AbapDeployConfig,
+    backendConfig?: FioriToolsProxyConfigBackend
+): AbapTarget {
+    let url, scp, client, destinationName, authenticationType;
+
+    if (isAppStudio() && destination) {
+        // the destination used during app generation
+        destinationName = destination.Name;
```

Comment:
Do we not need any other details here?

Reply:
Nope, the `AbapTarget` for destination only requires the name 

https://github.com/SAP/open-ux-tools/blob/13f99e6bbef880bb8b803f1e084a07d9c37b80e9/packages/system-access/src/types.ts#L14